frontend/scenes/StartScene.js

```diff
@@ -45,45 +46,26 @@ export default class StartScene extends Phaser.Scene {
   }
 
   create() {
-    this.cameras.main.setBackgroundColor("#FFFFC0");
-    this.logo = this.add
-      .image(
-        160,
-        160,
-        "logo"
-      ).setScale(0.3);
+    this.logo = this.add.image(160, 160, "logo").setScale(0.3);
 
-    this.enteredName = this.generateRandomName();
+    this.userName = this.generateRandomName();
 
     this.add
-      .image(
-        this.scale.width / 2,
-        this.scale.height / 2 + 115,
-        "createLobby"
-      )
+      .image(this.scale.width / 2, this.scale.height / 2 + 115, "createLobby")
       .setInteractive()
-      .on("pointerdown", () => {
-        if (this.enteredName.length > 0) {
-          this.createLobby(this.enteredName);
-        } else {
-          alert("Please enter a name");
-        }
-      });
+      .on("pointerdown", () => this.createLobby(this.userName));
 
     this.add
-      .image(
-        this.scale.width / 2,
-        this.scale.height - 50,
-        "joinLobby"
-      )
+      .image(this.scale.width / 2, this.scale.height - 50, "joinLobby")
       .setInteractive()
-      .on("pointerdown", () => {
-        if (this.enteredName.length > 0) {
-          this.joinLobby(this.enteredName);
-        } else {
-          alert("Please enter a name");
-        }
-      });
+      .on("pointerdown", () => this.joinLobby(this.userName));
+
+    this.errorMessage = new ErrorMessage(
+      this,
+      this.scale.width / 2,
+      this.scale.height / 2,
+      this.scale.width * 0.9,
+    );
```

Comment:
This scale object doesn't seem to get passed down into the ErrorMessage so it has to be calculated here